docs/content/preview/yugabyte-platform/alerts-monitoring/universe-logging.md

```diff
@@ -0,0 +1,82 @@
+- Configuring logging blocks other universe operations, such as backups and maintenance. Avoid changing your settings before maintenance windows and during scheduled backups. The operation will block a backup from running.
+- To limit performance impact and control costs, log and export only what you need. The default settings are based on best practices from open source PostgreSQL, the broader community, and YugabyteDB testing to ensure the impact is bounded and insignificant.
+
+## Database audit log
+
+To enable database audit logging for a universe, do the following:
+
+1. On the universe **Logs** tab, click **Enable Database Audit Logging**.
+
+    ![Configure Audit logging](/images/yp/log-export/configure-audit-logging.png)
+
+1. Select the YSQL statements to log.
+
+    - **Read** - SELECT and COPY when the source is a relation or a query.
+    - **Write** - INSERT, UPDATE, DELETE, TRUNCATE, and COPY when the destination is a relation.
+    - **Function** - Function calls and DO blocks.
+    - **Role** - Statements related to roles and privileges: GRANT, REVOKE, and CREATE/ALTER/DROP ROLE.
+    - **DDL** - All DDL that is not included in the ROLE class.
+    - **Misc** - Miscellaneous commands, such as DISCARD, FETCH, CHECKPOINT, VACUUM, and SET.
+
+1. Configure the [YSQL audit log settings](#ysql-audit-log-settings).
+
+1. Select the export configuration to use.
+
+1. Click **Enable and Export Database Audit Log**.
+
+YugabyteDB Anywhere begins the rolling restart.
+
+### YSQL audit log settings
+
+The YSQL audit logging settings are derived from the settings for logging used by the pgaudit extension. Statements are always logged.
+
+| Option | Description | Default |
+| :----- | :----- | :------ |
+| pgaudit.log_catalog | Log statements for the PostgreSQL system catalog relations in `pg_catalog`. These system catalog tables record system (as opposed to user) activity, such as metadata lookups and from third-party tools performing lookups.<br>These statements aren't required for typical auditing and you can disable this option to reduce noise in the log. | ON |
+| pgaudit.log_client | Enable this option to echo log messages directly to clients such as [ysqlsh](../../../admin/ysqlsh/) and psql. Log messages are printed directly to the shell, which can be helpful for debugging.<br>When enabled, you can set the level of logs that are output using `pgaudit.log_level`. | OFF |
+| pgaudit.log_level | Sets the [severity level](https://www.postgresql.org/docs/11/runtime-config-logging.html#RUNTIME-CONFIG-SEVERITY-LEVELS) of logs written to clients when `pgaudit.log_client` is on. Use this setting for debugging and testing.<br>Values: DEBUG1 .. DEBUG5, INFO, NOTICE, WARNING, LOG.<br>ERROR, FATAL, and PANIC are not allowed.<br>`pgaudit.log_level` only applies when `pgaudit.log_client` is on; otherwise the default LOG level is used. | LOG |
```

Comment:
Also need to mention setting the log_min_messages gflag for this pgaudit.log_level param to work properly. This is mentioned in the doc content file.